### DISCUSSION TITLE
[Question] Localize Package names in Cockpit UI

### DISCUSSION THREAD

**Discussion by DeadlySup** [cockpit-project/cockpit#19610] [Q&A]
Hello everyone,

after reading https://github.com/cockpit-project/cockpit/blob/main/doc/i18n.md and https://cockpit-project.org/guide/latest/packages I was able to properly localize the content of my cockpit package. However, I didn't find a way or indication how the name of the package itself could be localized. 

I also browsed some manifests for cockpit native apps (users, overview, etc.) and didn't find any indication that the labels were localized.

my manifest.json
````
{
	"version": 0,
	"tools": {
		"index": {
			"label": "Manual",
			"order": 6
		}
	},
	"content-security-policy": "default-src *; frame-src * 'unsafe-inline' 'unsafe-eval'"
}
````

I'd want a way to provide a defined translation similar to the po.xx.js.gz files to avoid the following browser translation behaviour
![image](https://github.com/cockpit-project/cockpit/assets/36188613/d74f8dc4-f51a-4875-8632-5cd47e8f3a96)

"Manuell" being German for doing something by hand (manual work) instead of the intended manual/handbook translation.

**Comment by jelly** [comment]
The menu entries can be translated, see for example Cockpit in German:

![image](https://github.com/cockpit-project/cockpit/assets/67428/8a8104ed-569b-4b81-88fc-1e1ceeee2727)

This should be translated using `manifest2po.js`, see our starter-kit makefile https://github.com/cockpit-project/starter-kit/blob/439fc281aa01441bf4695e838aea4e48097502f3/Makefile#L62
